Coding/fw/lib/WebServer/Pages.cpp

```diff
@@ -74,15 +76,15 @@ static String pageProcessor(const String &var)
 
     if (var == "STATION_SSID")
```

Comment:
you should use Yoda notation when comparing variables with constants. The constant should be on the left side, to trigger an error when you accidently type "=" instead of "==".